lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -972,12 +968,9 @@ static bool ContainsGenericTypeParameter(swift::Demangle::NodePointer node) {
 /// determine whether a node is generic or not, it needs to visit all
 /// nodes. The \p generic_walk argument specifies that the primary
 /// attributes have been collected and that we only look for generics.
-static uint32_t collectTypeInfo(TypeSystemSwiftTypeRef *module_holder,
-                                SwiftASTContext *swift_ast_context,
-                                swift::Demangle::Demangler &dem,
-                                swift::Demangle::NodePointer node,
-                                bool &unresolved_typealias,
-                                bool generic_walk = false) {
+uint32_t TypeSystemSwiftTypeRef::CollectTypeInfo(
+    swift::Demangle::Demangler &dem, swift::Demangle::NodePointer node,
+    bool &unresolved_typealias, bool generic_walk) {
```

Comment:
it appears `generic_walk` is always `false` and is never read. I would say feel free to remove it in this change.